activesupport/lib/active_support/continuous_integration.rb

```diff
@@ -0,0 +1,58 @@
+module ActiveSupport
+  class ContinuousIntegration
+    COLORS = {
+      banner: "\033[1;32m",   # Green
+      title: "\033[1;35m",    # Purple
+      subtitle: "\033[1;90m", # Medium Gray
+      error: "\033[1;31m",    # Red
+      success: "\033[1;32m"   # Green
+    }
+
+    def self.run(ci_declaration_path)
+      new.instance_eval(ci_declaration_path.read)
+    end
+
+    def initialize
+    end
+
+    def step(title, command)
+      echo :title, "\n\n#{title}"
+      echo :subtitle, "#{command}\n"
+
+      report(title) { system command }
```

Comment:
```suggestion
      report(title) { system { "CI" => "true" }, command }
```

If the goal is to behave like a CI, setting this env variable is preferable as some tools will behave differently, including Rails' default environment:  https://github.com/rails/rails/blob/c218fb6cd2bdd67771aac5bfce8097e43c05a26c/railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt#L16